lib/voice_manager.py

```diff
@@ -0,0 +1,102 @@
+"""
+voice関連の処理を掌握するモジュール
+"""
+import asyncio
+
+import discord
+
+class VoiceManager():
+    """
+    音声関連の処理をするクラス
+    """
+
+    def __init__(self):
+        """
+        初期化処理
+        """
+        self.member = None
+        self.before = None
+        self.after = None
+        self.is_join = None
+
+    async def diff_embed(self, member: discord.Member, before: discord.VoiceState, after: discord.VoiceState):
+        """
+        embedを送信する関数
+        Parameters
+        ----------
+        member: discord.Member
+            該当するメンバーののオブジェクト
+        before: discord.VoiceState
+            変更前のVoiceStateオブジェクト
+        after: discord.VoiceState
+            変更後のVoiceStateオブジェクト
+        """
+        self.member = member
+        self.before = before
+        self.after = after
+
+        if after.channel == before.channel or member.id in [684295118643265548]:
+            return
+        if after.channel is None:
+            self.is_join = False
+            await VoiceManager.KIKISEN_CHANNEL.send(embed=self.embed_generater())
+            await self.role_manager()
+            return
+        self.is_join = True
```

Comment:
```suggestion
        self.is_join = after.channel is not None
```
* ここで`is not`は`None`を否定しているわけではなくて`is not`で一つですぼくはPythonが嫌いです
  refer: https://qiita.com/tortuepin/items/44fdb63cc82dfd260575